azure_pipeline.yaml

```diff
@@ -6,18 +6,164 @@ trigger:
   branches:
     include:
     - master
-    
+  paths:
+    exclude:
+    - /modules/azure-landing-zone/*
+    - /components/global/*
+
 variables:
   terraformVersion: 0.12.26
   agentPool: 'ubuntu-18.04'
+  build: $(Build.BuildNumber)
+  isMaster: $[eq(variables['Build.SourceBranch'], 'refs/heads/master')]
   action:
 
+parameters:
+  - name: environments
+    type: object
+    default:
+    - deployment: 'sbox_cftapps_cluster_lb'
+      environment: 'sbox'
+      component: 'cftapps_cluster_lb'
+      service_connection: 'dcd-cftapps-sbox'
+      storage_account_rg: 'core-infra-sbox-rg'
+      storage_account_name: 'cftappssbox'
+      dependsOn: 'Precheck'
+
+    - deployment: 'sbox_cftapps_cluster_lb_backend'
+      environment: 'sbox'
+      component: 'cftapps_cluster_lb_backend'
+      service_connection: 'dcd-cftapps-sbox'
+      storage_account_rg: 'core-infra-sbox-rg'
+      storage_account_name: 'cftappssbox'
+      dependsOn: 'Precheck'
+
+    - deployment: 'sbox_shutter'
+      environment: 'sbox'
+      component: 'shutter'
+      service_connection: 'dcd-cftapps-sbox'
+      storage_account_rg: 'core-infra-sbox-rg'
+      storage_account_name: 'cftappssbox'
+      dependsOn: 'Precheck'
+
+    - deployment: 'sbox_compliance'
+      environment: 'sbox'
+      component: 'compliance'
+      service_connection: 'dcd-cftapps-sbox'
+      storage_account_rg: 'core-infra-sbox-rg'
+      storage_account_name: 'cftappssbox'
+      dependsOn: 'Precheck'
+    
+    # - deployment: 'ptl_dynatrace_activegate'
```

Comment:
why all this is commented out?

Reply:
Was running a quick test on some envs - all uncommented now.